### PR TITLE
Support when multiple scotties together cover the data stored in influx.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: common/api.go [hunk 1]
```diff
@@ -20,8 +20,9 @@ func NewInflux(influx config.Influx) (*Influx, error) {
 }
 
 // Query runs a query against this backend.
-func (d *Influx) Query(queryStr, epoch string) (*client.Response, error) {
-	return d.handle.Query(queryStr, d.data.Database, epoch)
+func (d *Influx) Query(
+	logger *log.Logger, query *influxql.Query, epoch string) (*client.Response, error) {
```
Comment (rgooch) [inline common/api.go]: Can you switch to the Logger interface type from github.com/Symantec/Dominator/lib/log ?
Also, I have a convention where the logger comes last in the parameter list.

File: common/common.go [hunk 2]
```diff
@@ -28,14 +30,18 @@ func (l *lastErrorType) Error() error {
...
 
 type queryerType interface {
-	Query(q, epoch string) (*client.Response, error)
+	Query(l *log.Logger, q *influxql.Query, epoch string) (
+		*client.Response, error)
 }
 
+// handleType represents a concrete server. Either a scotty or an influx db.
+// This interface exists to enable testing.
 type handleType interface {
```
Comment (rgooch) [inline common/common.go]: The Go convention with interface types is XYZ-er.

File: common/api.go (hunk 1 above)
Comment (keep94) [reply common/api.go]: Yep, I got a story for switching to lib/log.

**Comment by keep94** [comment]
Using Dominator/lib/log. Also s/handleType/dbQueryerType.

**Pull request merged**